mlrun/feature_store/retrieval/local_merger.py

```diff
@@ -35,10 +39,25 @@ def _generate_vector(
 
         feature_sets = []
         dfs = []
-        for name, columns in feature_set_fields.items():
+        keys = []
+        all_columns = list()
+
+        fs_link_list = self._create_linked_relation_list(
+            feature_set_objects, feature_set_fields
+        )
+
+        for node in fs_link_list:
+            name = node.name
```

Comment:
Isn't a lot of this code share with the rest of the mergers? Can't a lot of it move to the `BaseMerger` class?